pages/intro.js

```diff
@@ -0,0 +1,186 @@
+     */
+    const handleSave = async (type) => {
+        const textareas = document.getElementsByClassName(type)
+        const time = new Date(new Date().getTime())
+        let updateObj = {
+            WebsiteData: {
+                ...websiteData,
+                [type]: {
+                    editor: 'GRAB USER FROM REDUX?',
+                    time, 
+                    title: websiteData[type].title
+                }
+            }
+        }
+        let header, content
+        Array.prototype.forEach.call(textareas, (textarea) => {
+            if (textarea.style.resize == 'none') {
+                // if the textarea is the header (not resizable)
+                header = textarea.value
+                updateObj.WebsiteData[type]['header'] = header
+            } else {
+                // if text area is content (resizable)
+                content = textarea.value
+                updateObj.WebsiteData[type]['content'] = content
+            }
+        })
+        // TODO need to substitute the value at .doc(ID) with current website
+        await db.collection('Hackathons').doc('IntroTestHackathon')
+            .update(updateObj)
+            .then(() => {
+                setWebsiteData({
+                    ...websiteData,
+                    [type]: {
+                        ...websiteData[type],
+                        header,
+                        content,
+                        time
+                    }
+                })
+            })
+            .catch(() => {
+                alert('An error occured while trying to update the text section')
+            })
+        setIsEditingObj({
+            ...isEditingObj,
+            [type]: false
+        })
+    }
+
+    function Content(props) {
+        const { header, content, isEditing, type } = props
+        return (
+            <React.Fragment>
+                {
+                    isEditing ?
+                        <div style={{padding: '0px 40px 37px 40px'}}>
+                            <StyledHeader>Header</StyledHeader>
+                            <TextBox otherClassNames={type} defaultValue={header} />
+                            <StyledHeader>Body</StyledHeader>
+                            <TextBox otherClassNames={type} defaultValue={content} resize={true}/>
+                            <div style={{ marginTop: '27px', display: 'flex', float: 'right' }}>
+                                <StyledCancel onClick={() => handleCancel(type)}>
+                                    <p style={{ borderBottom: `2px solid ${COLOR.BLACK}`, margin: '0px' }}>
+                                        Cancel
+                                    </p>
+                                </StyledCancel>
+                                <Button onClick={() => handleSave(type)}>Save</Button>
+                            </div>
+                        </div>
+                        :
```

Comment:
Regarding my comment above about getting this information the react way:

You can add two `useState` hooks here that represents the header and the content. set an onChange event on the TextBoxs that will change those stateful values then pass them into save when you want to save them. That way you don't need to use getElementByClassName.